### PR TITLE
Monitor your apps: PHP and Node.js agents

### PR DESCRIPTION
**Pull request by akristen**
Adding 2 docs to match the "Monitor your Java app" doc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "remark-rehype": "^8.0.0",
     "rss": "^1.2.2",
     "sequelize": "^6.6.5",
+    "sharp": "^0.30.7",
```
Comment (roadlittledawn) [inline package.json:65]: when you ran that npm build command to fix `sharp` looks like you committed changes to updating sharp.

next time to fix those errors run:

```npm rebuild sharp --verbose```

can you uncommit the package.json and yarn.lock changes?

**Review by roadlittledawn — CHANGES_REQUESTED** [review]

File: src/content/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php.mdx [hunk 2]
```diff
@@ -19,6 +19,7 @@ redirects:
   - /docs/apm/agents/php-agent/getting-started
   - /docs/agents/php-agent
   - /docs/apm/agents/php-agent
+  - /docs/apm/agents/php-agent/installation/php-agent-installation-overview
```
Comment (roadlittledawn) [inline src/content/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php.mdx:22]: there might be some weird indentation for some of these redirects?

File: src/content/docs/apm/agents/nodejs-agent/getting-started/monitor-your-nodejs-app.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: Monitor your Node.js application
+tags:
+- Agents
+- Nodejs agent
+- Getting started
+metaDescription: 'For system requirements, installation and configuration procedures, and troubleshooting tips for New Relic''s Node.js agent, start here.'
+
```
Comment (roadlittledawn) [inline src/content/docs/apm/agents/nodejs-agent/getting-started/monitor-your-nodejs-app.mdx:8]: may want to remove empty line here

File: package.json (hunk 1 above)
Comment (akristen) [reply package.json]: It's asking me to send the local build to a different address. Should I try to run `npm rebuild sharp --verbose` to see if updating (?) that way will set everything correct? Gatsby logs say there are no errors, but is pinging something as deprecated.

Comment (akristen) [reply package.json]: spoilers i did that ^ anyway and nothing resolved. i'll ping your team tomorrow e_e

**Pull request merged**